src/hooks/useAdKeywordsSocketListener.jsx

```diff
@@ -9,23 +9,33 @@ const useAdKeywordsSocketListener = (videoId) => {
   const userId = useUserId();
 
   useEffect(() => {
-    socket.on("connect", () => {
+    if (!userId) return;
```

Comment:
```suggestion
    if (!userId) {
      return;
    }
```
중괄호 넣어주면 더 좋을 것 같아요!

Reply:
감사합니다!
중괄호 반영했습니다. ☺️